sno/diff.py

```diff
@@ -226,6 +255,11 @@ def diff_with_writer(
     is_flag=True,
     help="Make the program exit with codes similar to diff(1). That is, it exits with 1 if there were differences and 0 means no differences.",
 )
+@click.option(
+    "--srs",
```

Comment:
Can we DRY the option, since presumably we'll end up using it everywhere?

Reply:
At present only in diff and show, and it's quite tiny. Happy to DRY it if it gets bigger or we start using it in a few more places (where do you have in mind?)

Reply:
spatial indexing, query, export, vtiles, import, load/replace, checkout...

Reply:
can DRY it when we do some of those.